infra/templates/az-hello-world/README.md

```diff
@@ -0,0 +1,112 @@
+# The Azure Hello World Pattern
+
+The `az-hello-world` template is intended to be a reference for running a single public Linux Container within an Azure Application Service Plan.
+
+> *Have you completed the quick start guide? Deploy your first infrastructure as code project with Cobalt by following the [quick-start guide](https://github.com/microsoft/cobalt/blob/master/docs/2_QUICK_START_GUIDE.md).*
+
+## Use-Case
+
+This particular template creates an Azure environment with our smallest infrastructure footprint and is the recommended template highlighted in our [quick-start guide](https://github.com/microsoft/cobalt/blob/master/docs/2_QUICK_START_GUIDE.md). It's a practical first step.
+
+## Provisioned Resources
+
+This deployment creates the following:
+
+ 1. Azure Resource Group
+ 2. Linux App Service Plan
+ 3. App Service Container w/ public i.p.
+
+> ![image](https://user-images.githubusercontent.com/10041279/66533295-0fc64b80-ead8-11e9-96c0-ea5441cde854.png)
+
+## Intended audience
+
+Application developer that is brand new to Cobalt templating and it's *Cobalt Infrastructure Template* (CIT) developer workflow.
+
+## Prerequisites
+
+* App dev experience (prev. "infrastructure as code" experience not required)
+* An Azure Subscription
+  * [Azure Portal](https://portal.azure.com/) - This template needs to deploy infrastructure within an Azure subscription.
+* An Azure Service Principal
+  * [Azure Service Principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal) - This template needs permissions to deploy infrastructure within an Azure sbuscription.
+* An Azure Storage Account
+  * [Azure Storage Account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview) - An account for tracking terraform remote backend state. You can use our backend state setup [template](../infra/templates/backend-state-setup/README.md) to provision the storage resources.
+* Terminal with bash shell
+  * [WSL](https://code.visualstudio.com/docs/remote/wsl) or [Git Bash](https://git-scm.com/downloads) - The shell environment needed to follow along with the provided instructions.
+* Local environment
+  * [Local environment setup](https://github.com/microsoft/cobalt/tree/master/test-harness#local-environment-setup)
```

Comment:
out of date

Reply:
Local environment setup is no longer reference and this is mainly for testing which is a different part of the walk through.